.openshift-ci/gcp/command.sh

```diff
@@ -59,4 +70,4 @@ source .openshift-ci/env.sh
 source .openshift-ci/google-cloud-sdk/install.sh
 source .openshift-ci/google-cloud-sdk/init.sh
 
-main "kernel-packer-osci" $COMMAND
+main "kernel-packer-osci" "${GCP_SSH_KEY_USER}" $COMMAND
```

Comment:
is `GCP_SSH_KEY_USER` populated by the OSCI pipeline?

Reply:
Yep.